src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const objCalculator = {
+    operate(callback, number) {
+      callback(number);
+
+      return this;
+    },
+    add(number) {
+      objCalculator.result += number;
+    },
+    subtract(number) {
+      objCalculator.result -= number;
```

Comment:
The `subtract` method correctly modifies the `result` property. Ensure that the `operate` method, if intended to use callbacks for arithmetic operations, aligns with this approach.